cumulus_library/template_sql/sql_utils.py

```diff
@@ -201,24 +195,31 @@ def denormalize_complex_objects(
 
 
 def validate_schema(
-    cursor: databases.DatabaseCursor,
-    schema: str,
+    database: databases.DatabaseBackend,
     expected_table_cols: dict,
-    parser: databases.DatabaseParser,
```

Comment:
The data metrics study currently does some of this itself:
```
        for table, cols in fields_to_check.items():
            query = base_templates.get_column_datatype_query(schema, table.lower(), cols.keys())
            cursor.execute(query)
            table_schema = cursor.fetchall()
            self.schemas[table] = parser.validate_table_schema(cols, table_schema)
```

- But I can switch it to the sql_utils methods, if we think those are more stable/public. (and it would help get fixes like this empty table thing)